components/match_ticker/commons/match_ticker.lua

```diff
@@ -334,6 +339,13 @@ end
 ---@param matches table[]
 ---@return table[]
 function MatchTicker:filterMatches(matches)
+	-- Remove matches with wrong region
+	if self.config.regions then
+		matches = Array.filter(matches, function(match)
+			return Table.includes(self.config.regions, match.region)
```

Comment:
would we want to add support for multi region events?
```suggestion
			return Array.any(match.regions, function(region) return Table.includes(self.config.regions, region))
```

Reply:
Multi-region events would cause issues with the tournament ticker's way of filtering, so probably not